### PR TITLE
Add readme about application.

### LINKED ISSUE #11: Add readme about how to submit a profile application?
With descriptions for each field and other notes help user understand the process.

### PR DESCRIPTION
**Pull request by furoxr**
Fixes #11

### CODE REVIEW

File: Readme.md
Comment (wuminzhe) [inline Readme.md:9]: how many approves does it need?

Comment (furoxr) [reply Readme.md:9]: Issues only require approval from one maintainer, but Pull Requests can be configured to require approval from multiple maintainers before they can be merged.

**Review by hackfisher — APPROVED** [review]

File: Readme.md
Comment (hackfisher) [inline Readme.md]:
```suggestion
This repository serves as a storage for application info of Darwinia Community DAO Profiles NFTs. In addition, it provides a streamlined process for contributors to apply and automatically receive their NFTs.
```

**Review by hackfisher — APPROVED** [review]

**Pull request merged**
